### PR TITLE
Add Test

### PR DESCRIPTION
**Pull request by Hopding**
Added the test for the `Authorize` lambda that was removed in https://github.com/maryvilledev/codesplainUI/pull/297.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "babel": {
+    "presets": [ "es2015" ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/maryvilledev/codesplain-lambdas.git"
+  },
+  "author": "Maryville Dev Center",
+  "license": "SEE LICENSE IN LICENSE.md",
```
Comment (dane-johnson) [inline package.json]: This should be just the name of the license (i.e. `GNU GPL`).

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "codesplain-lambdas",
+  "version": "0.1.0",
```
Comment (dane-johnson) [inline package.json]: Is this correct?

**Comment by thebho** [comment]
Saving this for next release

File: package.json (hunk 2 above)
Comment (Hopding) [inline package.json]: That is the version we are using for [codesplainUI](https://github.com/maryvilledev/codesplainUI/blob/master/package.json#L3) and we are still in beta, so it is correct as far as I'm aware.

Comment (thebho) [inline package.json]: The versioning is done on the tag/S3 path level, so it's not necessary in the lambdas, so I don't think it matters how we handle it in the package.json.

**Pull request merged**